docs/version0.44.md

```diff
@@ -76,6 +77,16 @@ The `jcmd` tool also now supports specifying `0` as a VMID to target all VMs.
 
 For more information, see [Java diagnostic command (`jcmd`) tool](tool_jcmd.md).
 
+### Flag names that refer to the fields of `J9BuildFlags` changed in the DDR code
+
+The Direct Dump Reader (DDR) code enables reading system dump data by using the OpenJ9 Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool. To read the build flags in the system dump data, the DDR code has flag names that refer to `J9BuildFlags`.
+
+Earlier, some flag names in `J9BuildFlags` were mapped back to the flags in `j9.flags`. Such a mapping made providing support to all system dumps complex because some flag names in `J9BuildFlags` were different at different times. Now, when `J9BuildFlags` is generated for each build, the flag names are those names that are specified in `j9cfg.h` (derived from `j9cfg.h.ftl` or `j9cfg.h.in`) instead of the names that are defined in `j9.flags`. Code that has references to flag names that are defined in `j9.flags` results in compilation errors.
+
+Therefore, the flag names that are used in the DDR code to refer to the fields of `J9BuildFlags` are changed. The flag names are now those names that are in `j9cfg.h`. For example, `env_data64` is now referred to as `J9VM_ENV_DATA64`.
+
+You can extend the DDR code, adding your own commands, by writing plug-ins. If the user plug-ins contain code that has `J9BuildFlags` references to read the build flags in the system dump data, you must use the names that are specified in `j9cfg.h` as flag names in the plug-in code also.
+
```

Comment:
Suggest rewriting this completely:
```
### Field names in `J9BuildFlags` have changed

The Direct Dump Reader (DDR) code enables reading system dump data by using the OpenJ9
Diagnostic Tool Framework for Java (DTFJ) API or the [`jdmpview`](tool_jdmpview.md) tool.
To facilitate access to build flags in the system dump data, DDR code uses fields of `J9BuildFlags`.
The names of those fields have varied over time making it challenging to support all system dumps.

Earlier, field names in `J9BuildFlags` were based on names defined in `j9.flags`.
Now, when the `J9BuildFlags` is generated for each build, the flag names are those
names that are specified in `j9cfg.h` (derived from `j9cfg.h.ftl` or `j9cfg.h.in`)
instead of the names that are defined in `j9.flags`.
For example, `env_data64` is now referred to as `J9VM_ENV_DATA64`.

You can extend the DDR code, adding your own commands, by writing plug-ins.
If the user plug-in code contains references to fields of `J9BuildFlags` to read the build flags
in the system dump data, those references must change to use the names as specified in `j9cfg.h`.
```